guide/02-api-overview/release_notes_241.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<ol><li>Typo in the new parameter for Folder.add(): "upload_file_size"</li><li> <a href="https://github.com/ArcGIS/geosaurus/pull/12487" target="_blank">https://github.com/ArcGIS/geosaurus/pull/12487</a> - for arcgis.gis.WorkflowManager, the new parameter "centralized_data_references" can be added to release note in create_diagram() and update_diagram()</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thanks, Shaung.</p><ol><li>Corrected it.</li><li>Added note regarding using cdr's in both methods</li></ol><p> </p>

Reply:
1.
![image](https://github.com/user-attachments/assets/9c309ac2-43b6-461e-903a-08d435395691)

2.
![image](https://github.com/user-attachments/assets/a7736604-54fb-4ffe-9da6-29cd0cbd0dfe)